tornadox/sqrt.py

```diff
@@ -60,7 +60,7 @@ def update_sqrt(transition_matrix, cov_cholesky):
             [zeros_bottomleft.T, zeros_bottomright.T],
         ]
     )
-    big_triu = jax.scipy.linalg.qr(blockmat, mode="r", pivoting=False)
+    (big_triu,) = jax.scipy.linalg.qr(blockmat, mode="r", pivoting=False)
```

Comment:
Doesn't it return `(Q,R)`? If yes:
```suggestion
    (,big_triu) = jax.scipy.linalg.qr(blockmat, mode="r", pivoting=False)
```

Reply:
"Not returned if mode='r'." Got it.